src/infra/core/src/repos/mod.rs

```diff
@@ -7,68 +7,52 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-mod object_repository_local_fs;
-pub use object_repository_local_fs::*;
-
-mod object_repository_http;
-pub use object_repository_http::*;
-
-mod object_repository_s3;
-pub use object_repository_s3::*;
-
-mod object_repository_in_memory;
-pub use object_repository_in_memory::*;
-
-mod named_object_repository_local_fs;
-pub use named_object_repository_local_fs::*;
-
+mod dataset_factory_impl;
+mod dataset_impl;
+mod dataset_repository_helpers;
+mod dataset_repository_local_fs;
+mod dataset_repository_s3;
+mod metadata_block_repository_caching_inmem;
+mod metadata_block_repository_helpers;
+mod metadata_block_repository_impl;
+mod metadata_chain_impl;
+mod metadata_chain_visitor;
+mod metadata_chain_visitor_impl;
```

Comment:
Let's call this module `metadata_chain_validators`, as we will not be putting any other visitors in there.

Reply:
Done